bentoml/serve.py

```diff
@@ -393,3 +393,202 @@ def serve_production(
+def serve_runner(
+    bento_identifier: str,
+    working_dir: str,
+    runner_name: str,
+    port: int | None = None,
+    host: str | None = None,
+    backlog: int = Provide[BentoMLContainer.api_server_config.backlog],
+) -> None:
+    """
+    Experimental API for serving a BentoML runner.
+    """
+    working_dir = os.path.realpath(os.path.expanduser(working_dir))
+    svc = load(bento_identifier, working_dir=working_dir, standalone_load=True)
+
+    from circus.sockets import CircusSocket  # type: ignore
+    from circus.watcher import Watcher  # type: ignore
+
+    watchers: t.List[Watcher] = []
+    circus_socket_map: t.Dict[str, CircusSocket] = {}
+    uds_path = None
+
+    ensure_prometheus_dir()
+
+    with contextlib.ExitStack() as port_stack:
+        for runner in svc.runners:
+            if runner.name == runner_name:
+                if port is None:
+                    port = port_stack.enter_context(reserve_free_port())
+                if host is None:
+                    host = "127.0.0.1"
+                circus_socket_map[runner.name] = CircusSocket(
+                    name=runner.name,
+                    host=host,
+                    port=port,
+                    backlog=backlog,
+                )
+
+                watchers.append(
+                    Watcher(
+                        name=f"runner_{runner.name}",
+                        cmd=sys.executable,
+                        args=[
+                            "-m",
+                            SCRIPT_RUNNER,
+                            bento_identifier,
+                            "--runner-name",
+                            runner.name,
+                            "--fd",
+                            f"$(circus.sockets.{runner.name})",
+                            "--working-dir",
+                            working_dir,
+                            "--no-access-log",
+                            "--worker-id",
+                            "$(circus.wid)",
+                        ],
+                        copy_env=True,
+                        stop_children=True,
+                        use_sockets=True,
+                        working_dir=working_dir,
+                        numprocesses=runner.scheduled_worker_count,
+                    )
+                )
+                break
+        else:
+            raise ValueError(f"Runner {runner_name} not found in {bento_identifier}")
+
+    arbiter = create_standalone_arbiter(
+        watchers=watchers,
+        sockets=list(circus_socket_map.values()),
+    )
+
+    try:
+        arbiter.start(
+            cb=lambda _: logger.info(  # type: ignore
+                'Starting RunnerServer from "%s"\n running on http://%s:%s (Press CTRL+C to quit)',
+                bento_identifier,
+                host,
+                port,
+            ),
+        )
+    finally:
+        if uds_path is not None:
+            shutil.rmtree(uds_path)
+
+
+@inject
+def serve_bare_api_server(
```

Comment:
"Serve" isn't technically accurate here. How about just `start_api_server` and `start_runner`?